.travis.yml

```diff
@@ -12,19 +12,26 @@ env:
     # PG_VERSION (docker) and PG_GIT_TAG (Git) should be the same
     - PG_VERSION=9.6.6 PG_GIT_TAG=REL9_6_6
     - PG_VERSION=10.2 PG_GIT_TAG=REL_10_2
+    - EXEC_BACKEND="-DEXEC_BACKEND=1" PG_VERSION=10 PG_GIT_TAG=REL_10_2
+    - EXEC_BACKEND="-DEXEC_BACKEND=1" PG_VERSION=9.6 PG_GIT_TAG=REL9_6_6
 before_install:
   # We need the PostgreSQL source for running the standard PostgreSQL
   # regression tests
   - git clone --branch ${PG_GIT_TAG} --depth 1 https://github.com/postgres/postgres.git /tmp/postgres
-  - docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres postgres:${PG_VERSION}-alpine
+  - if [[ -z "$EXEC_BACKEND" ]]; then docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres postgres:${PG_VERSION}-alpine; fi
+  - if [[ -n "$EXEC_BACKEND" ]]; then docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres timescaledev/postgres_exec_backend:${PG_VERSION}-alpine; fi
```

Comment:
Is there a reason to not just build this at `timescaledev/postgres-dev` with the tag `exec_backend-pg9.6-alpine` and `exec_backend-pg10-alpine` (or similar, not sure the OS is needed)? That was my initial intention, it doesn't seem like we need a separate repo for each since they are fundamentally the same 'code', just different flavors which seems more worthy of tags.

Reply:
Also they're categorized based on what I think will update together:
pgindent will basically never update unless we switch formatting tools, exec_backend is only a stopgap until we get windows testing fixed, and we'll eventually have versions of the postgres images built with DEBUG in there own repository. It seems a shame to clutter up that one, which has other legitimate uses with the more idiosyncratic images.

Reply:
I think pgindent is fine, but I don't really see the need to create the additional repos when basically these are all variations of postgres, just with different options. That seems more like a job for tags than separate repositories. We can also remove tags if we ever remove/deprecate certain images.

Reply:
updating to `timescaledev/postgresdev:exec_backend-${PG_VERSION}-alpine`. for some reason I have the feeling the `alpine` will be important later.